ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -211,6 +206,30 @@ then
 	if bmi160 start
 	then
 	fi
+
+	# Start either ICM2060X or BMI055. They are both connected to the same SPI bus and use the same
```

Comment:
We seem to be missing the:
```
	# Internal SPI bus ICM-20602-G is rotated 90 deg yaw
 	if mpu6000 -R 2 -T 20602 start
 	then
 	fi
 ```
Do we expect to have a mix of sensors? Some BMI and some ICM and 9250? 
If not we can speed up start up with one probe and [use this pattern](https://github.com/scheiermanns/Firmware/blob/308d4ea61139b6a740132ce0393cb368ec30f04c/ROMFS/px4fmu_common/init.d/rc.sensors#L113-L123) to wrap the BMI startups